Improve docs (#1242)
imgbot/Imgbot: Docs/configuration.md

> Optionally, for more advanced scenarios, you can configure Imgbot using an `.imgbotconfig` json file in the root of your GitHub repository <!-- exact filename? can we remove the json? --> and set to your liking.

Author: question as comment

Author: ```suggestion
Optionally, for more advanced scenarios, you can configure Imgbot using an `.imgbotconfig` file in the root of your GitHub repository. The file contains the configuration in JSON format.
```

> Optionally, for more advanced scenarios, you can configure Imgbot using an `.imgbotconfig` json file in the root of your GitHub repository <!-- exact filename? can we remove the json? --> and set to your liking.

Author: ```suggestion
Optionally, for more advanced scenarios, you can configure Imgbot using an `.imgbotconfig` json file in the root of your GitHub repository and set to your liking.
```

This is not a required step to using Imgbot and is only for more advanced scenarios.
Optionally, for more advanced scenarios, you can configure Imgbot using an `.imgbotconfig` json file in the root of your GitHub repository <!-- exact filename? can we remove the json? --> and set to your liking.

This file should be placed in the root of the repository and set to your liking.

> See [this past issue](https://github.com/dabutvin/Imgbot/issues/49) for details about the location for this file.

Author: I checked the issue and could not find out how it helps users to understand why or why not the file is placed in the root folder.

}
```

Outside of the `.imgbotconfig` file, you can configure additional options by logging into [imgbot.net](https://imgbot.net/app). This is the current list of settings supported in this UI:

 - Default branch override (Imgbot will look after a different branch instead of the repository default branch)

> For any other option you would like to see supported, feel free to [open an issue](https://github.com/dabutvin/Imgbot/issues/new) or shoot an email over

Author: shorter

Author: ```suggestion
If you'd like to see any other configuration settings supported, [open an issue](https://github.com/dabutvin/Imgbot/issues/new) or email us 
```
